taxi/forms.py

```diff
@@ -0,0 +1,34 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Car, Driver
+
+
+class LicenseNumberFormMixin(forms.Form):
+    license_number = forms.CharField()
+
+
+class DriverCreationForm(UserCreationForm, LicenseNumberFormMixin):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name", "last_name", "license_number"
+        )
+
+
+class DriverLicenseUpdateForm(LicenseNumberFormMixin, forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
The queryset for the `drivers` field in `CarForm` should be filtered to include only drivers. Currently, it uses `get_user_model().objects.all()`, which may include non-driver users. Consider filtering the queryset to only include users with the `Driver` role or model.